docker/convert.go

```diff
@@ -53,7 +53,7 @@ func ConvertToAPI(s *Service) (*ConfigWrapper, error) {
 	return &result, nil
 }
 
-func volumes(c *project.ServiceConfig, ctx *Context) map[string]struct{} {
+func volumes(c *project.ServiceConfig, ctx project.Context) map[string]struct{} {
```

Comment:
Gotcha, I thought it was odd we'd have a passing build but with a function signature change